commands/create.go

```diff
@@ -511,15 +527,15 @@ func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, userdataFlag, customInst
 		return err
 	}
 	// Remove the shebang
-	customScriptContent = regexp.MustCompile(`^#!.*\n`).ReplaceAll(customScriptContent, nil)
+	modifiedCustomScriptContent := regexp.MustCompile(`^#!.*\n`).ReplaceAll(customScriptContent, nil)
```

Comment:
Encode the custom script contents here and pass it to `replaceUserdataFile`. You won't have to pass `customScriptContent` to that function.